pebble_tool/sdk/project.py

```diff
@@ -49,6 +49,11 @@ def _sanity_check(self):
                 or not os.path.exists(os.path.join(self.project_dir, 'wscript')):
             raise OutdatedProjectException("This project is very outdated, and cannot be handled by this SDK.")
 
+        if int(self.sdk_version) > 3:
```

Comment:
This assumes cast-to-int is possible, which isn't a valid assumption (and is why this is a string in the first place).